src/reducers/auth.js

```diff
@@ -0,0 +1,41 @@
+import api from 'utils/api';
+
+const LOGIN_START = 'rmp/users/LOGIN_START';
+const LOGIN_SUCCESS = 'rmp/users/LOGIN_SUCCESS';
+const LOGIN_FAILURE = 'rmp/users/LOGIN_FAILURE';
+
+const initialState = {
+  isLoggedIn: false,
+  isLoggingIn: false,
+  error: '',
+};
+
+export const loginUser = (service, userData) => async dispatch => {
+  dispatch({ type: LOGIN_START });
+  try {
+    localStorage.setItem('token', userData.accessToken);
+    localStorage.setItem('service', service);
```

Comment:
just a suggestion - we can use indexed db, so its not blocking and we can simply use promises?

Reply:
Local storage is fine for this purpose.